CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Change Log
 
+## [1.3.12]
+
+* Fix: debug attach not working with go app
+* Fix: detect errors in yaml file before actually parse it with node-yaml-parser lib
+* Fix: retrieve tool paths outside kubernetes object and update README
+* Feat: handle predefined vscode variables in extension setting
+* Feat: add setting to suppress helm not found warning
+* Fix: handle ANSI color codes in log webview
```

Comment:
I think we can remove the `Fix:` and `Feat:` prefixes from the change log list. I added them because i used to use conventional commits but maybe it's more readable if we remove them from the change log?